app/models/route.rb

```diff
@@ -0,0 +1,12 @@
+class Route < ApplicationRecord
+  has_many :trips, dependent: :nullify
+
+  validates :source, :destination, presence: true, format: { with: /\A[A-Za-z\s]+\z/ }
+  attr_accessor :name
```

Comment:
what represents attr_accessor?

Reply:
Bhaia, `attr_accessor` is a shortcut method when you need both `attr_reader` and `attr_writer`.